### PR TITLE
Updated to compile for the Arduino Pro Mini. Resolved compiler warnings.

### PR DESCRIPTION
**Pull request by rimim**
I also removed the function prototypes because if you have separate function declaration and definitions then the compiler believes that default arguments should only be in the declaration and spits out a bunch of warnings. The prototypes aren't necessary in the Arduino sketch environment.

### CODE REVIEW

File: PSIPro.ino [hunk 1]
```diff
@@ -979,7 +973,7 @@ void scanColLeftRight(unsigned long time_delay, int start_row, CRGB color, bool
 // Display a matrix using the byte array.  Colours are defined so that
 // if the matrix has a 1, we use fgcolor, and 0 is bgcolor.
 // Optionally colors 2,3,4,5,6,7,8 (allowing a total of nine colours to be used in a pattern.
-void displayMatrixColor(byte PROGMEM * matrix, CRGB fgcolor, CRGB bgcolor, bool displayMe, unsigned long runtime, 
+void displayMatrixColor(const byte /*PROGMEM*/ * matrix, CRGB fgcolor, CRGB bgcolor, bool displayMe, unsigned long runtime,
```
Comment (nhutchison) [inline PSIPro.ino]: Rather than commenting this out, let's just remove it.

File: PSIPro.ino [hunk 2]
```diff
@@ -435,7 +429,7 @@ void loop()
     }
     else
     {
-      lastPSIeventCode == defaultPattern;
+      lastPSIeventCode = defaultPattern;
```
Comment (nhutchison) [inline PSIPro.ino:432]: Nice catch.

**Review by nhutchison — COMMENTED** [review]
Only change I'd request is to remove the commented out PROGMEM from displayMatrixColor

File: PSIPro.ino (hunk 1 above)
Comment (rimim) [reply PSIPro.ino]: Done.

**Review by nhutchison — APPROVED** [review]

**Pull request merged**